src/flint/fmpq.jl

```diff
@@ -713,6 +713,8 @@ convert(::Type{fmpq}, a::Integer) = fmpq(a)
 
 convert(::Type{fmpq}, a::fmpz) = fmpq(a)
 
+Base.convert(::Type{fmpq}, x::Rational) = fmpq(x.num, x.den)
```

Comment:
Our Base imports are in Nemo/src/Nemo.jl

On 13 September 2016 at 01:12, Jorge Fernández de Cossío Díaz <
notifications@github.com> wrote:

> In src/flint/fmpq.jl
> <https://github.com/wbhart/Nemo.jl/pull/62#discussion_r78471874>:
>
> > @@ -713,6 +713,8 @@ convert(::Type{fmpq}, a::Integer) = fmpq(a)
> >
> >  convert(::Type{fmpq}, a::fmpz) = fmpq(a)
> >
> > +Base.convert(::Type{fmpq}, x::Rational) = fmpq(x.num, x.den)
>
> I'm not sure. I did not find any import Base.convert.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/wbhart/Nemo.jl/pull/62/files/25c9cb65f332a4f014fa3a04a8938d85ab7198e7#r78471874>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAOzpnQD48yM5YJdsV1jhffxQYQicgFiks5qpdxagaJpZM4J6u5P>
> .
>